pyproject.toml

```diff
@@ -4,10 +4,10 @@ build-backend = "poetry.core.masonry.api"
 
 [tool.poetry]
 name = "pyscript-cli"
-version = "0.2.0"
+version = "0.3.0"
```

Comment:
I'd arguably downgrade that version to 0.0.1 😆  .. Not that it matters too much right now. We should probably have a conversation on that and align at some point though.

Reply:
Strictly speaking, semver doesn't allow `0` as a minor version. But I think this can be safely made as a patch.

This is one reason why having a separate repo for the CLI makes sense. We don't necessarily have to align the version numbers. Still, we should probably stay on `0.2.*` and make lots of patches as-needed.

Unfortunately I jumped the gun a bit and rolling back to `0.1.*` would break things unless I pulled a release from PyPI.